### PR TITLE
Not all pkgs have -test.changes so deal with it

### PR DESCRIPTION
**Pull request by asdil12**
Fixes #6

### CODE REVIEW

File: os-autoinst-obs-auto-submit [hunk 1]
```diff
@@ -24,7 +24,9 @@ update_package() {
     sed -i -e 's,mode="buildtime" mode="disabled",mode="buildtime",' _service
     # ignore those updates
     rm -f _service:*-test.changes
-    cp .osc/*-test.changes .
+    if test -f .osc/*-test.changes; then
```
Comment (okurz) [inline os-autoinst-obs-auto-submit]: I don't think this can work as expected. Trying out locally `test -f *` or anything that returns multiple files fails

Comment (asdil12) [reply os-autoinst-obs-auto-submit]: fixed.

**Comment by okurz** [comment]
thx. Let's see if this works :)

**Comment by asdil12** [comment]
Looks like as if it got fixed: https://build.opensuse.org/request/show/778077

**Pull request merged**